src/chrome/content/rules/heritage.gov.hk.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="heritage.gov.hk">
+	<target host="heritage.gov.hk" />
+	<target host="www.heritage.gov.hk" />
+
+	<rule from="^http://(?:www\.)?heritage\.gov\.hk/"
```

Comment:
`(?:www\.)` and `(www\.)` don't have difference here. I suggest to remove the `?:`